CODEOWNERS

```
@@ -1 +1,6 @@
-* @buildpacks/core-team
+/platform.md @buildpacks/implementation-team-lead
+/buildpack.md @buildpacks/bat-team-lead
+/distribution.md @buildpacks/distribution-team-lead
+/extensions/buildpack-registry.md @buildpacks/distribution-team-lead
+/extensions/project-descriptor.md @buildpacks/platform-team-lead
+* @buildpacks/team-leads
```

```suggestion
* @buildpacks/team-leads
* @buildpacks/toc 
```

Should we add the TOC here as reviewers but not blockers?

---

Re-reading the codeowners syntax - I believe if we add toc here it will add them as blockers. I am going to revert this change and I will capture it in the readme to add TOC as reviewers.